telethon_examples/auto_reply.py

```diff
@@ -47,12 +48,10 @@ def __init__(self):
                 code = input('Enter the auth code: ')
                 try:
                     code_ok = self.sign_in(user_phone, code)
-                except RPCError as e:
+                except SessionPasswordNeededError:
                     if e.password_required:
```

Comment:
You're using `SessionPasswordNeededError`, but that doesn't have any `.password_required` field! I got rid of the field in favor of the `SessionPasswordNeededError` exception itself.

Reply:
Agh, I could swear I committed that change.

Reply:
No problem, being able to squash from GitHub is nice.